hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/AssignmentManager.java

```diff
@@ -2280,24 +2295,48 @@ private void addToPendingAssignment(final HashMap<RegionInfo, RegionStateNode> r
    * For system tables, we must assign them to a server with highest version.
    */
   public List<ServerName> getExcludedServersForSystemTable() {
+    return getExcludedServersForSystemTable(false);
+  }
+
+  /**
+   * Get a list of servers that this region can not assign to.
```

Comment:
There doesn't seem to be a 'region' passed to this method.... So is it any system table region?